src/main/java/com/bookstore/resources/BookResource.java

```diff
@@ -0,0 +1,146 @@
+package com.bookstore.resources;
+
+import com.bookstore.model.Book;
+import com.bookstore.service.BookServiceDao;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+import javax.ws.rs.core.Response;
+
+@Path("/books")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class BookResource {
+    BookServiceDao bookServiceDao = new BookServiceDao();
```

Comment:
Since you created an interface `BookService` you could have specified that as the type instead of the `BookServiceDao` implementation class